README.md

```diff
@@ -613,8 +633,89 @@ Birdcache methods are:
       audience={"username": ["salih", "sam", "timo"]},
       soundTemplate="openup"
     )
+
+### `Lexi` resource <a name = "lexi"> </a>
+
+Lexi is a dictionary that will replace problematic words with phonetic spellings. For example,ensuring the city name `reading` is pronounced correctly. Dictionaries are split into languages and types. For example, the city `reading` would be of type `location` with language `en-gb` and in this case in the `UkCities69` dictionary.
```

Comment:
Do we want to say that we using phonetic spelling?

Reply:
Im not sure I guess this is PO question

Reply:
@LindaKivine